actors/stage/src/main/java/com/ea/orbit/actors/Stage.java

```diff
@@ -241,6 +246,18 @@ public void setMode(final StageMode mode)
         hosting.start();
         execution.start();
 
+        try
+        {
+            Class.forName("com.ea.orbit.metrics.MetricsManager"); //make sure the metrics manager is on the classpath.
+
+            MetricsManager.getInstance().initializeMetrics(metricsConfig);
```

Comment:
Yeh, it won't be super pretty but you should be able to achieve it with reflection.
That's the only major problem I see with the code, otherwise it looks great!